build-tools/packages/build-tools/src/common/typeValidatorConfig.ts

```diff
@@ -0,0 +1,29 @@
+/*!
```

Comment:
Would be nice to move type validator out of fluid-build in the future.

Reply:
I have work scheduled this sprint to do it.